source/_docs/hsts.md

```diff
@@ -130,6 +130,9 @@ Once you've installed the module or plugin you plan to use, you should immediate
 
 How you configure or include these attributes raises the rigor of the security that your HSTS effort provides. [Here is a great overview of how and why to use the above noted attributes](https://hstspreload.org/){.external}.
 
+## HSTS Before the Application Level
+The configuration described above sets the HSTS header when the CMS loads. If you need HSTS to be set before that, consider a DNS-level implementation from a third-party provider like [CloudFlare](https://support.cloudflare.com/hc/en-us/articles/204183088-Understanding-HSTS-HTTP-Strict-Transport-Security-){.external}
```

Comment:
@alexfornuto should this be DNS-level or CDN-level?

Reply:
@carl-alberto can you confirm?

Reply:
@alexfornuto sorry I'm not sure :( , might be CDN or both

but it seems this PR will be overruled if this feature is implemented and rolled out https://github.com/pantheon-systems/documentation/pull/4047 so I think we can close this as there is an upcoming platform feature that can implement this instead of using 3rd party or Cloudflare

Reply:
@carl-alberto #4047 has been on hold since September. I can't get any sort of ETA. I think we should proceed as if it will never go through until we hear differently.